rowhammer_tester/scripts/version.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+from rowhammer_tester.scripts.utils import RemoteClient
+
+if __name__ == "__main__":
+    wb = RemoteClient()
+    wb.open()
+
+    # Dump CSR register containing git revision
+    for name, reg in wb.regs.__dict__.items():
```

Comment:
We could simplify this part to just `wb.regs.gitrev_hash.read()`.

Reply:
Applied